web/src/components/PoiListItem.tsx

```diff
@@ -1,65 +1,42 @@
+import Avatar from '@mui/material/Avatar'
 import Divider from '@mui/material/Divider'
+import ListItem from '@mui/material/ListItem'
+import ListItemAvatar from '@mui/material/ListItemAvatar'
+import ListItemButton from '@mui/material/ListItemButton'
+import ListItemText from '@mui/material/ListItemText'
+import Typography, { TypographyProps } from '@mui/material/Typography'
 import { styled } from '@mui/material/styles'
 import React, { ReactElement } from 'react'
 import { useTranslation } from 'react-i18next'
 
 import { PoiModel } from 'shared/api'
 
 import { PoiThumbnailPlaceholder } from '../assets'
-import { helpers } from '../constants/theme'
-import Button from './base/Button'
+import Link from './base/Link'
 
-const ListItemContainer = styled('ul')`
-  font-family: ${props => props.theme.legacy.fonts.web.contentFont};
-  display: flex;
-  padding: clamp(10px, 1vh, 20px) 0;
-  cursor: pointer;
+const StyledListItemButton = styled(ListItemButton)`
+  align-items: flex-start;
+  gap: ${props => props.theme.spacing(2)};
+` as typeof ListItemButton
 
-  &:first-of-type {
-    padding-top: 0;
+const StyledListItemAvatar = styled(ListItemAvatar)`
+  [class*='MuiAvatar-root'] {
+    margin-top: 8px;
+    width: 94px;
+    height: 94px;
```

Comment:
How about changing this to 64:
The design says `56px` I feel that the ratio of the design isn't right.
so I picked something slightly above: 
```suggestion
    width: 64px;
    height: 64px;
```
And no need to `margin-top: 8px;`

Reply:
I would say it looks better when they are on the same level and it would be better to keep the margin there :thinking: